Response/JWTAuthenticationSuccessResponse.php

```diff
@@ -28,18 +28,8 @@
      */
     public function __construct($token, array $data = null)
     {
-        $this->token = $token;
+        $data = is_null($data) ? [] : $data;
```

Comment:
Could `$response = null` be changed to `$response = []` in the signature? It would remove the need for this line